php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java

```diff
@@ -1136,6 +1139,21 @@ public void visit(ConstantDeclaration node) {
         }
     }
 
+    private int computeConstNodeNameLength(ConstantDeclaration node) {
+        int modifierLength = 0;
+        if (!Modifier.isImplicitPublic(node.getModifier())) {
+            modifierLength = node.getModifierString().length() + " ".length(); // NOI18N
```

Comment:
Here and below - isn't `" ".length()` simply one? I believe that JVM is smart enough and new strings are not being created over and over, but still. The same applies for `"const ".length()`.

Perhaps we could define these lengths as constants? That could be nicely readable, I believe.

Reply:
Fixed: https://github.com/apache/netbeans/compare/cce07cb62abab658d6447f7a25af1aebbfe5cf79..f586da1b03743c847cdc63fb37307acbc912ec2f